Jenkinsfile

```diff
@@ -289,7 +291,7 @@ pipeline {
                         }
                     }
                 }
-            }
+            } // parallel
```

Comment:
This comment should be before `}` as you commented `{`

Reply:
@LukaszKwasniewski change to indicate where parallel would close?

Reply:
Yes, I've indicated were the parallel section ends.
@grzegorz-robak it is not commented out because the `parallel {` section at the top had to be replaced with `stages {` Jenkinsfiles does not allow two `stage {` sections one after another